### PR TITLE
Add link to contributor ladder

### PR DESCRIPTION
**Pull request by xmulligan**
Add a link from the maintainers doc to the contributor ladder will help people understand how to grow in the project towards becoming a maintainer

```release-note
Add link in maintainers.md and contributing guide to contributor ladder
```

### CODE REVIEW

**Review by tklauser — APPROVED** [review]

File: Documentation/contributing/development/contributing_guide.rst
Comment (xmulligan) [inline Documentation/contributing/development/contributing_guide.rst]:
```suggestion
To help contributors grow in both privileges and responsibilities for the
```

**Comment by tklauser** [comment]
@xmulligan please squash the third commit (the spelling fix) into the second commit.

File: Documentation/contributing/development/contributing_guide.rst
Comment (qmonnet) [inline Documentation/contributing/development/contributing_guide.rst:412]: Why “also”?

**Review by qmonnet — CHANGES_REQUESTED** [review]
> Adding a link from the contributing guide to the contributor ladder will help people discover it and move up level in the project

There is already a link to the ladder in this guide. The new thing is the dedicated paragraph.

Looks good otherwise, thanks!

File: Documentation/contributing/development/contributing_guide.rst
Comment (xmulligan) [reply Documentation/contributing/development/contributing_guide.rst:412]: I would say the contributor ladder is a guide to grow in the project, but not the only way. People could become committers without ever becoming reviewers. Not likely, but possible

Comment (qmonnet) [reply Documentation/contributing/development/contributing_guide.rst:412]: Hmm OK fair enough, although that's not obvious from the text.
I'm not gonna block the PR any longer on this though :+1:

**Review by qmonnet — APPROVED** [review]
Thanks!

**Pull request merged**
